google/cloud/aiplatform/pipeline_jobs.py

```diff
@@ -0,0 +1,351 @@
+        pipeline_spec["runtimeConfig"] = runtime_config
+
+        _set_enable_caching_value(pipeline_spec["pipelineSpec"], enable_caching)
+
+        if encryption_spec_key_name is not None:
+            pipeline_spec["encryptionSpec"] = {"kmsKeyName": encryption_spec_key_name}
+
+        if labels:
+            if not isinstance(labels, Dict):
+                raise ValueError(
+                    "Expect labels to be a mapping of string key value pairs. "
+                    'Got "{}" of type "{}"'.format(labels, type(labels))
+                )
+            for k, v in labels.items():
+                if not isinstance(k, str) or not isinstance(v, str):
+                    raise ValueError(
+                        "Expect labels to be a mapping of string key value pairs. "
+                        'Got "{}".'.format(labels)
+                    )
+
+            pipeline_spec["labels"] = labels
+
+        self._gca_resource = gca_pipeline_job_v1beta1.PipelineJob(
+            display_name=display_name,
+            pipeline_spec=pipeline_spec,
+            labels=labels,
+            runtime_config=None,
+            encryption_spec=initializer.global_config.get_encryption_spec(
+                encryption_spec_key_name=encryption_spec_key_name
+            ),
+        )
+
+    @base.optional_sync()
+    def run(
+        self,
+        service_account: Optional[str] = None,
+        network: Optional[str] = None,
+        sync: bool = True,
+    ) -> None:
+        """Run this configured PipelineJob.
+        Args:
+            service_account (str):
+                Optional. Specifies the service account for workload run-as account.
+                Users submitting jobs must have act-as permission on this run-as account.
+            network (str):
+                Optional. The full name of the Compute Engine network to which the job
+                should be peered. For example, projects/12345/global/networks/myVPC.
+                Private services access must already be configured for the network.
+                If left unspecified, the job is not peered with any network.
+            sync (bool):
+                Whether to execute this method synchronously. If False, this method
+                will unblock and it will be executed in a concurrent Future.
+        """
+
+        if service_account:
+            self._gca_resource.pipeline_spec.service_account = service_account
+
+        if network:
+            self._gca_resource.pipeline_spec.network = network
+
+        _LOGGER.log_create_with_lro(self.__class__)
+
+        self._gca_resource = self.api_client.select_version(
+            _PIPELINE_CLIENT_VERSION
+        ).create_pipeline_job(parent=self._parent, pipeline_job=self._gca_resource)
+
+        _LOGGER.log_create_complete_with_getter(
+            self.__class__, self._gca_resource, "pipeline_job"
+        )
+
+        _LOGGER.info("View Pipeline Job:\n%s" % self._dashboard_uri())
```

Comment:
Is the info log available on stdout? Just want to make sure users will see this message after submitting a job.

Aslo, there's a minor difference from the KFP client. KFP client renders an HTML for this in the notebook (IPython) environment.

Reply:
I believe this is configured in L42 `logging.basicConfig(level=logging.INFO, stream=sys.stdout)`

Reply:
> We should do this as across the urls produced in the notebook instead of a one off.

That's reasonable.